griffin-doc/deploy/deploy-guide.md

```diff
@@ -105,7 +105,7 @@ You should also modify some configurations of Griffin for your environment.
 	# elasticsearch.password = password
     ```
 
-- <b>measure/src/main/resources/env.json</b>
+- <b>measure/src/main/resources/env-streaming.json</b>
```

Comment:
@bhlx3lyx7 Just updated it. Thanks!

Reply:
@bhlx3lyx7 One more...
Is name of configuration file for spark job has changed from `sparkJob`? Because it isn't exists and there are only `sparkProperties`...but it seems not defining spark job. Could you look on it?